### PR TITLE
Create workflow to add primer/behaviors issues to the Primer teams backlog for triage

### PR DESCRIPTION
**Pull request by lesliecdubs**
I noticed recently that issues filed in the `primer/behaviors` repo don't end up in our inbox for triage (e.g. https://github.com/orgs/github/projects/4503), so are likely getting missed. 

This PR attempts to create an "add to inbox" workflow for this repo so that new and reopened issues will get labelled as `react` and `rails` (so we can discuss at the appropriate weekly maintainer syncs) and moved into the Inbox of the [Primer teams backlog](https://github.com/orgs/github/projects/4503).

Questions:
- Are all of the `vars`, `steps`, and `secrets` used in this workflow valid? I assumed these values are defined in the specified actions, but would be good to confirm.
- Any other ideas for a better triage plan for `primer/behaviors` issues?

### CODE REVIEW

**Review by pksjce — APPROVED** [review]

File: .github/workflows/add-to-inbox.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      - name: Add rails label to issue
+        run: |
+          gh issue edit $ISSUE_URL --add-label rails
+        env:
+          GH_TOKEN: ${{ steps.get-primer-access-token.outputs.token }}
+      - name: Add react label to issue
+        run: |
+          gh issue edit $ISSUE_URL --add-label react
+        env:
+          GH_TOKEN: ${{ steps.get-primer-access-token.outputs.token }}
```
Comment (joshblack) [inline .github/workflows/add-to-inbox.yml:28]: Just wanted to share a couple of options to refactor if they'd be helpful!

I think `--add-label` supports multiple labels through a comma:

```suggestion
      - name: Add labels to issue
        run: |
          gh issue edit $ISSUE_URL --add-label 'rails,react'
        env:
          GH_TOKEN: ${{ steps.get-primer-access-token.outputs.token }}
```

Another idea was to just do the command one after the other if we want them separate:

```suggestion
      - name: Add rails label to issue
        run: |
          gh issue edit $ISSUE_URL --add-label rails
          gh issue edit $ISSUE_URL --add-label react
        env:
          GH_TOKEN: ${{ steps.get-primer-access-token.outputs.token }}
```

**Review by joshblack — APPROVED** [review]

**Comment by joshblack** [comment]
@lesliecdubs made sure the variables and secrets are available so should be good to go now!

**Review by camertron — APPROVED** [review]
Looks like @jonrohan added the necessary variables + secrets, so we should be good to merge this 👍

**Comment by lesliecdubs** [comment]
Thanks all ✨

File: .github/workflows/add-to-inbox.yml (hunk 1 above)
Comment (lesliecdubs) [reply .github/workflows/add-to-inbox.yml]: > I think `--add-label` supports multiple labels through a comma:

TIL! Thanks for simplifying. Looks like the merged code follows this suggestion: https://github.com/primer/behaviors/pull/239/commits/06cef459c90f3afbe841d003c5db3f710a08a0b4

**Pull request merged**